Index: chromium-129.0.6668.58/third_party/angle/src/libANGLE/Constants.h
===================================================================
--- chromium-129.0.6668.58.orig/third_party/angle/src/libANGLE/Constants.h
+++ chromium-129.0.6668.58/third_party/angle/src/libANGLE/Constants.h
@@ -9,6 +9,7 @@
 #ifndef LIBANGLE_CONSTANTS_H_
 #define LIBANGLE_CONSTANTS_H_
 
+#include <cstddef>
 #include "common/platform.h"
 
 #include <stddef.h>
