Index: chromium-128.0.6613.113/content/browser/BUILD.gn
===================================================================
--- chromium-128.0.6613.113.orig/content/browser/BUILD.gn
+++ chromium-128.0.6613.113/content/browser/BUILD.gn
@@ -246,7 +246,6 @@ source_set("browser") {
     "//services/viz/privileged/mojom",
     "//services/viz/public/cpp/gpu",
     "//services/viz/public/mojom",
-    "//services/webnn:webnn_service",
     "//services/webnn:webnn_switches",
     "//services/webnn/public/mojom",
     "//skia",
Index: chromium-128.0.6613.113/content/gpu/BUILD.gn
===================================================================
--- chromium-128.0.6613.113.orig/content/gpu/BUILD.gn
+++ chromium-128.0.6613.113/content/gpu/BUILD.gn
@@ -75,7 +75,6 @@ target(link_target_type, "gpu_sources")
     "//services/service_manager/public/mojom",
     "//services/tracing/public/cpp",
     "//services/viz/privileged/mojom",
-    "//services/webnn:webnn_service",
     "//services/webnn/public/mojom",
     "//skia",
     "//third_party/angle:angle_gpu_info_util",
