Index: chromium-126.0.6478.57/components/viz/service/BUILD.gn
===================================================================
--- chromium-126.0.6478.57.orig/components/viz/service/BUILD.gn
+++ chromium-126.0.6478.57/components/viz/service/BUILD.gn
@@ -284,7 +284,6 @@ viz_component("service") {
     "//services/metrics/public/mojom",
     "//services/tracing/public/cpp:cpp",
     "//services/viz/privileged/mojom",
-    "//services/webnn:webnn_service",
     "//skia",
     "//skia:skcms",
     "//third_party/libyuv",
Index: chromium-126.0.6478.57/components/viz/service/gl/gpu_service_impl.cc
===================================================================
--- chromium-126.0.6478.57.orig/components/viz/service/gl/gpu_service_impl.cc
+++ chromium-126.0.6478.57/components/viz/service/gl/gpu_service_impl.cc
@@ -902,15 +902,6 @@ void GpuServiceImpl::BindClientGmbInterf
 void GpuServiceImpl::BindWebNNContextProvider(
     mojo::PendingReceiver<webnn::mojom::WebNNContextProvider> pending_receiver,
     int client_id) {
-  if (!main_runner_->BelongsToCurrentThread()) {
-    main_runner_->PostTask(
-        FROM_HERE,
-        base::BindOnce(&GpuServiceImpl::BindWebNNContextProvider, weak_ptr_,
-                       std::move(pending_receiver), client_id));
-    return;
-  }
-  webnn::WebNNContextProviderImpl::Create(std::move(pending_receiver),
-                                          GetContextState(), gpu_feature_info_);
 }
 #endif  // !BUILDFLAG(IS_CHROMEOS)
 
Index: chromium-126.0.6478.57/content/browser/BUILD.gn
===================================================================
--- chromium-126.0.6478.57.orig/content/browser/BUILD.gn
+++ chromium-126.0.6478.57/content/browser/BUILD.gn
@@ -227,7 +227,6 @@ source_set("browser") {
     "//services/viz/privileged/mojom",
     "//services/viz/public/cpp/gpu",
     "//services/viz/public/mojom",
-    "//services/webnn:webnn_service",
     "//services/webnn:webnn_switches",
     "//skia",
     "//skia/public/mojom",
Index: chromium-126.0.6478.57/content/gpu/BUILD.gn
===================================================================
--- chromium-126.0.6478.57.orig/content/gpu/BUILD.gn
+++ chromium-126.0.6478.57/content/gpu/BUILD.gn
@@ -75,7 +75,6 @@ target(link_target_type, "gpu_sources")
     "//services/service_manager/public/mojom",
     "//services/tracing/public/cpp",
     "//services/viz/privileged/mojom",
-    "//services/webnn:webnn_service",
     "//services/webnn/public/mojom",
     "//skia",
     "//third_party/angle:angle_gpu_info_util",
Index: chromium-126.0.6478.57/services/BUILD.gn
===================================================================
--- chromium-126.0.6478.57.orig/services/BUILD.gn
+++ chromium-126.0.6478.57/services/BUILD.gn
@@ -58,7 +58,6 @@ test("services_unittests") {
       "//services/viz/public/cpp/compositing:tests",
       "//services/viz/public/cpp/gpu:tests",
       "//services/viz/public/cpp/hit_test:tests",
-      "//services/webnn:tests",
     ]
     if (is_ios) {
       bundle_deps =
