Index: chromium-130.0.6723.44/sandbox/linux/BUILD.gn
===================================================================
--- chromium-130.0.6723.44.orig/sandbox/linux/BUILD.gn
+++ chromium-130.0.6723.44/sandbox/linux/BUILD.gn
@@ -304,6 +304,12 @@ if (is_linux || is_chromeos) {
       # These files have a suspicious comparison.
       # TODO fix this and re-enable this warning.
       "-Wno-sign-compare",
+
+      "-fPIE",
+    ]
+
+    ldflags = [
+      "-pie",
     ]
   }
 }
