Index: chromium-128.0.6613.36/chrome/browser/extensions/BUILD.gn
===================================================================
--- chromium-128.0.6613.36.orig/chrome/browser/extensions/BUILD.gn
+++ chromium-128.0.6613.36/chrome/browser/extensions/BUILD.gn
@@ -936,6 +936,7 @@
     "//components/resources",
     "//components/safe_browsing:buildflags",
     "//components/safe_browsing/content/browser/web_ui:web_ui",
+    "//components/safe_browsing/content/common/proto:download_file_types_proto",
     "//components/safe_browsing/core/browser/db:database_manager",
     "//components/safe_browsing/core/common",
     "//components/safe_browsing/core/common:safe_browsing_prefs",
