Index: chromium-129.0.6668.58/build/rust/cargo_crate.gni
===================================================================
--- chromium-129.0.6668.58.orig/build/rust/cargo_crate.gni
+++ chromium-129.0.6668.58/build/rust/cargo_crate.gni
@@ -429,7 +429,7 @@ template("cargo_crate") {
           rust_abi_target,
         ]
       }
-      if (current_cpu == "arm64" || current_cpu == "x64") {
+      if (current_cpu == "arm64" || current_cpu == "x64" || current_cpu == "ppc64") {
         args += [
           "--pointer-width",
           "64",
